teos/src/api/http.rs

```diff
@@ -217,6 +218,15 @@ async fn get_subscription_info(
     Ok(reply::with_status(body, status))
 }
 
+async fn ping(addr: Option<SocketAddr>) -> Result<impl Reply, Rejection> {
+    log::debug!(
+        "Received a ping request from {}",
+        addr.map_or("an unknown address".to_owned(), |a| a.to_string())
+    );
+    Ok(Response::builder().body("").into_response())
```

Comment:
I've forgot about this one.

```suggestion
    Ok(reply::reply())
```

Reply:
This also means that 

`use warp::http::Response;` won't be necessary.

Reply:
That's cool!!